website/docs/advanced/team-management/saml/identity-providers/onelogin.md

```diff
@@ -0,0 +1,149 @@
+---
+id: onelogin
+title: OneLogin
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Connect ConfigCat with OneLogin via SAML.
+
+### Introduction
+Each SSO Identity Provider requires specific information to configure a SAML integration. The following guide will walk you through on how you can connect ConfigCat with OneLogin as a SAML Identity Provider.
+
+### 1. Create an Application in OneLogin
+
+- Log in to <a href="https://app.onelogin.com/login" target="_blank">OneLogin</a>, and select `Applications`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/onelogin/applications.png" />
+
+- Click on `Add App`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/onelogin/add_app.png" />
+
+- Type `saml` into the search bar, and select `SAML Custom Connector (Advanced)`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/onelogin/select_app.png" />
+
+- Enter a descriptive `Display Name`, then click `Save`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/onelogin/app_name.png" />
+
+The next step will guide you on how to collect the information required for the appearing `Configuration` page.
+
+### 2. Configure SAML for the OneLogin Application
+- Open your organization's authentication settings on the <a href="https://app.configcat.com/organization/authentication" target="_blank">ConfigCat dashboard</a>.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/dashboard/authentication.png" />
+
+- Select the domain you want to configure with SAML, and click `Set` under the `SAML SSO status`.
+
+  <img class="saml-tutorial-img" src="/static/assets/saml/dashboard/domains.png" />
+
+- From the appearing dialog, copy the following values and paste them into the OneLogin application's configuration page.
+    - Copy `Entity ID` and paste it into the `Audience (EntityID)`.
+    - Copy `Assertion Consumer Service` and paste it into the `ACS (Consumer) URL`.
+    - Paste the same `Assertion Consumer Service` into the `ACS (Consumer) URL Validator` in regex format.
```

Comment:
Fú, ez elég combos. Vajon nem tudnánk valahogy segíteni?

Reply:
Hat, nekem is ki kellett szopkodnom az eterbol, nem igazan van jo leiras errol, de a screenshoton ott a pelda legalabb, csak a vegen a guid-ot kell lecserelni